chirp/drivers/tdh8.py

```diff
@@ -1254,20 +1254,7 @@ def get_block_xy(self, number):
 
         return ([x_list, y_list])
 
-    def set_memory(self, mem):
-        _mem = self._get_mem(mem.number)
-        _nam = self._get_nam(mem.number)
-
-        if mem.empty:
-            _mem.set_raw("\xff" * 16)
-            if self.MODEL in H8_LIST:
-                _nam.set_raw("\xff" * 16)
-            else:
-                _nam.set_raw("\xff" * 8)
-            return
-
-        _mem.set_raw("\x00" * 16)
-
```

Comment:
Okay so the fix is to do the (ridiculous) flag setting below all the time, and *then* mark out the memory and bail after that right?

Can we name it something other than "evaluate_usedflags". It's not really evaluating anything, it's setting, updating, etc.

Also, when I was reviewing this driver I tried to help the submitter make it less crazy than the if-per-bit thing. Can we refactor that as part of this? I can stack something on top here if you want.

Reply:
Saw this after typing my shpeil:

Yes. Simple fix, elusive problem.

Sure. I'll think on it

Sure. It aggrivated me too. I can do that quick.